### PR TITLE
Remove api imports (only on store, non-context)

### PR DESCRIPTION
**Pull request by jacogr**
cc @kwingram25 Would like your eyes on this.

### CODE REVIEW

File: packages/app-contracts/src/Deploy.tsx [hunk 1]
```diff
@@ -256,8 +256,6 @@ class Deploy extends ContractModal<Props, State> {
     if (record) {
       const address = record.event.data[1] as any as AccountId;
 
-      await apiUi.isReady;
```
Comment (jacogr) [inline packages/app-contracts/src/Deploy.tsx]: No need, components are only rendered when API is ready

File: packages/app-contracts/src/store.ts [hunk 2]
```diff
@@ -30,8 +30,6 @@ class Store extends EventEmitter {
   }
 
   async saveCode (codeHash: string | Hash, partial: Partial<CodeJson>) {
-    await api.isReady;
```
Comment (jacogr) [inline packages/app-contracts/src/store.ts]: Not needed here, only needed in loadAll which can happen before we are ready.

File: packages/app-contracts/src/Deploy.tsx (hunk 1 above)
Comment (kwingram25) [reply packages/app-contracts/src/Deploy.tsx]: Got it!

**Comment by jacogr** [comment]
Yes. As soon as the api is ready it gets added. And since the app waits for the api before rendering (well apart from settings), it will be there.

**Comment by jacogr** [comment]
Just the store needs some magic, sometimes - at least in load. Just to make 100% sure there since it is not gated in the same way, so technically can be used from anywhere.

**Pull request merged**